TriblerGUI/tribler_window.py

```diff
@@ -603,6 +606,15 @@ def on_add_torrent_browse_file(self):
             [self.pending_uri_requests.append(u"file:%s" % filename) for filename in filenames[0]]
             self.process_uri_request()
 
+    def on_add_mdblob_browse_file(self):
+        filenames = QFileDialog.getOpenFileNames(self,
+                                                 "Please select the .mdblob file",
+                                                 QDir.homePath(),
+                                                 "Tribler metadata files (*.mdblob)")
+        if len(filenames[0]) > 0:
+            [self.pending_uri_requests.append(u"file:%s" % filename) for filename in filenames[0]]
```

Comment:
I would prefer a more explicit `for` statement here, instead of using list comprehensions without using the resulting list.

Reply:
yep